_maps/shuttles/arrival_box.dmm

```diff
@@ -78,18 +78,8 @@
 /turf/open/floor/plating,
 /area/shuttle/arrival)
 "p" = (
-/obj/structure/chair,
-/turf/open/floor/mineral/titanium/blue,
-/area/shuttle/arrival)
-"q" = (
-/obj/structure/chair,
-/obj/machinery/light/small{
-	dir = 4
-	},
-/turf/open/floor/mineral/titanium/blue,
-/area/shuttle/arrival)
-"r" = (
-/obj/structure/chair{
+/obj/structure/chair/shuttle{
+	icon_state = "shuttle_chair";
```

Comment:
You can find&replace all of these lines away, since this var is set by the `/shuttle` subtype already.